engine/launcher/src/main/scala/org/enso/launcher/installation/DistributionInstaller.scala

```diff
@@ -0,0 +1,344 @@
+package org.enso.launcher.installation
+
+import java.nio.file.Files
+
+import org.enso.cli.CLIOutput
+import org.enso.launcher.FileSystem.PathSyntax
+import org.enso.launcher.internal.OS
+import org.enso.launcher.internal.installation.DistributionManager
+import org.enso.launcher.{
+  FileSystem,
+  GlobalConfigurationManager,
+  InternalOpts,
+  Logger
+}
+
+import scala.util.control.NonFatal
+
+class DistributionInstaller(
+  manager: DistributionManager,
+  autoConfirm: Boolean,
+  bundleActionOption: Option[DistributionInstaller.BundleAction]
+) {
+  final private val installed = manager.LocallyInstalledDirectories
+  private val env             = manager.env
+
+  /**
+    * Names of additional files that are not essential to running the
```

Comment:
Your doc formatting isn't quite right! You need to have a blank line between the summary and the body. Please check all your doc comments for this!